bashbrew/bashbrew.sh

```diff
@@ -62,16 +62,21 @@ build options:
 push options:
   --no-push          Don't push, print what would push
 
+version options:
+  --no-datestamp	 Don't create datestamped image
+
 EOUSAGE
 }
 
 # arg handling
-opts="$(getopt -o 'h?' --long 'all,docker:,help,library:,logs:,namespaces:,no-build,no-clone,no-push,src:' -- "$@" || { usage >&2 && false; })"
+opts="$(getopt -o 'h?' --long 'all,docker:,help,library:,logs:,namespaces:,no-build,no-clone,no-push,src:,no-datestamp' -- "$@" || { usage >&2 && false; })"
 eval set -- "$opts"
 
 doClone=1
 doBuild=1
 doPush=1
+doDatestamp=1
+dateStamp=$(date +'%F')
```

Comment:
Let's use the same convention that docker uses which is the current date without dashes like so `20151125`. Passing the `--utc` flag would also be a good idea